Endpoint without prefix error

Currently END_POINT_without_prefix in api.dart is api.junto.foundation 

Running into the following issue here and there. Can you examine? cc @Nash0x7E2 

`flutter: [PerspectivesBloc.mapEventToState]: Fetching perspectives
flutter: Error during fetching perspectives
flutter: SocketException: Failed host lookup: 'api.junto.foundation' (OS Error: nodename nor servname provided, or not known, errno = 8)
flutter: #0      IOClient.send (package:http/src/io_client.dart:33:24)
flutter: <asynchronous suspension>
flutter: #1      BaseClient._sendUnstreamed (package:http/src/base_client.dart:176:38)
flutter: #2      BaseClient.get (package:http/src/base_client.dart:35:7)
flutter: #3      JuntoHttp.get (package:junto_beta_mobile/utils/junto_http.dart:49:23)
flutter: <asynchronous suspension>
flutter: #4      UserServiceCentralized.getUserPerspective (package:junto_beta_mobile/backend/services/user_service.dart:98:22)
flutter: #5      UserRepo.getUserPerspective (package:junto_beta_mobile/backend/repositories/user_repo.dart:31:25)
flutter: #6      PerspectivesBloc._fetchUserPerspectives (package:junto_beta_mobile/screens/collective/perspectives/bloc/perspectives_bloc.dart:66:35)
flutter: #7      PerspectivesBloc._mapFetchToState (package:junto_beta_mobile/screens/collective/perspectives/bloc/perspectives_bloc.dart:50:27)
flutter: <asynchronous suspension>
flutter: #8      PerspectivesBloc.mapEventToState (package:junto_beta_mobile/screens/collective/perspectives/bloc/perspectives_bloc.dart:31:14)
flutter: <asynchronous suspension>
flutter: #9      Bloc._bindStateSubject.<anonymous closure> (package:bloc/src/bloc.dart:155:14)
flutter: #10     Stream.asyncExpand.onListen.<anonymous closure> (dart:async/stream.dart:576:30)
flutter: #11     _rootRunUnary (dart:async/zone.dart:1134:38)
flutter: #12     _CustomZone.runUnary (dart:async/zone.dart:1031:19)
flutter: #13     _CustomZone.runUnaryGuarded (dart:async/zone.dart:933:7)
flutter: #14     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
flutter: #15     _DelayedData.perform (dart:async/stream_impl.dart:594:14)
flutter: #16     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:710:11)
flutter: #17     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:670:7)
flutter: #18     _rootRun (dart:async/zone.dart:1122:38)
flutter: #19     _CustomZone.run (dart:async/zone.dart:1023:19)
flutter: #20     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
flutter: #21     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:965:23)
flutter: #22     _rootRun (dart:async/zone.dart:1126:13)
flutter: #23     _CustomZone.run (dart:async/zone.dart:1023:19)
flutter: #24     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
flutter: #25     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:965:23)
flutter: #26     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
flutter: #27     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
flutter:
flutter: [ERROR] [PrintLogger.logException]: Caught error: SocketException: Failed host lookup: 'api.junto.foundation' (OS Error: nodename nor servname provided, or not known, errno = 8)`

--- a/lib/backend/services/user_service.dart
+++ b/lib/backend/services/user_service.dart
@@ -67,7 +67,7 @@ class UserServiceCentralized implements UserService {
     final String authKey = _prefs.getString('auth');
 
     final Uri _uri = Uri.http(
-      END_POINT_without_prefix,
+      END_POINT,
       '/users',
       _buildQueryParam(param, queryType),
     );

--- a/lib/utils/junto_http.dart
+++ b/lib/utils/junto_http.dart
@@ -44,7 +44,7 @@ class JuntoHttp {
 
   Future<http.Response> get(String resource,
       {Map<String, String> headers, Map<String, String> queryParams}) async {
-    final Uri _uri = Uri.http(
+    final Uri _uri = Uri.https(
         END_POINT_without_prefix, '/$kServerVersion$resource', queryParams);
     return httpClient.get(
       _uri,
@@ -56,7 +56,7 @@ class JuntoHttp {
       {Map<String, String> headers,
       Map<String, String> queryParams,
       dynamic body}) async {
-    final Uri _uri = Uri.http(
+    final Uri _uri = Uri.https(
         END_POINT_without_prefix, '/$kServerVersion$resource', queryParams);
     return httpClient.patch(
       _uri,

